### ISSUE TITLE
Internet Explorer 11 SCRIPT438:  Object doesn't support property or method from

### ISSUE THREAD

**Issue by hanbar** [bokuweb/react-rnd#299]
When trying to resize IE throws error from this function:

```
var toConsumableArray = function (arr) {
  if (Array.isArray(arr)) {
    for (var i = 0, arr2 = Array(arr.length); i < arr.length; i++) arr2[i] = arr[i];

    return arr2;
  } else {
    return Array.from(arr);
  }
};
```

and does not resize the div.

**Comment by jeff-wolff** [comment]
Fixed this with a shim from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Polyfill

Performance is still ass in IE11

**Comment by bokuweb** [comment]
@hanbar @go299 @jeff-wolff I've fixed this issue by this PR https://github.com/bokuweb/re-resizable/pull/283. Could you please try latest version if you get a chance :)

**Comment by bokuweb** [comment]
fixed by bokuweb/re-resizable#283
